packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -111,3 +98,21 @@ abstract class Screen {
     return null;
   }
 }
+
+mixin OfflineScreenMixin<T extends StatefulWidget, U> on State<T> {
+  bool loadingOfflineData = false;
```

Comment:
make this a getter and private field. That way there is no way a caller can accidentally set the value.

Reply:
done